docs/remote/tunnels.md

```diff
@@ -164,4 +166,4 @@ As mentioned in the [`code` CLI doc](/docs/editor/command-line.md#create-remote-
 
 ### Can I use other Remote Development Extensions or a dev container while I'm tunneling?
 
-Currently, you can connect to WSL over Remote - Tunnels.
+Yes! Currently, you can connect to [WSL](/docs/remote/wsl.md) and [dev containers](/docs/devcontainers/containers.md#open-a-folder-on-a-remote-tunnel-host-in-a-container) over Remote - Tunnels.
```

Comment:
We should mention dev containers now works here too! (Only on vscode desktop right now, vscode.dev support is still on the way)

Reply:
Yes agree! Wondering if the current mention of dev containers sounds good, or if you think we should change it?

Reply:
Oh, I see we have docs for that, but this heading says "Can I use other Remote Development Extensions or a dev container while I'm tunneling?" and the answer is now 'yes to both'

Reply:
Ah, so maybe while my current phrasing here lists WSL and dev containers in one sentence, it'd instead be clearer to separate them into 2 bullet points or spaced out sentences (where we can expand a bit upon each, such as the dev containers web vs desktop limitations you mention)?

Or were you mostly suggesting rephrasing the "Yes" to "Yes to both" to start off the section being more explicit?

Apologies if I'm still missing your main suggestion for this section!

Reply:
Oh, my bad, my brain read everything after `[WSL]` to the tunnel as one link and I skipped over that dev containers was mentioned 🤦  lgtm